docs/core-library/filters.md

```diff
@@ -38,10 +38,10 @@ Unix Command Line Examples
 
 * output all indexes within distance 1 of the `H3Index` 845ad1bffffffff
 
-     `echo 845ad1bffffffff | kRing 1`
+     `echo 845ad1bffffffff | kRing -k 1`
```

Comment:
Alternately this could be written as `kRing -k 1 --origin 845ad1bffffffff` to not use pipes

Reply:
Fixed.